docs/content/concepts.mdx

```diff
@@ -268,6 +305,45 @@ In the configuration of the document type, we have four relations: `viewer`, `co
 
 For more information, please see [Direct Access](./modeling/direct-access.mdx).
 
+</details>
+
+<details>
+<summary>
+
+## What Is A Directly Related User Type?
+
+A **directly related user type** is an array specified in the type definition to indicate what types of users can be directly related to that relation.
+
+</summary>
+
+For the following model, only [relationship tuple](#what-is-a-relationship-tuple) with [user](#what-is-a-user) [type](#what-is-a-type) `user` may be assigned to document.
+
+<AuthzModelSnippetViewer
+  configuration={{
+    type_definitions: [
+      {
+        type: 'document',
+        relations: {
+          viewer: {
+            this: {},
+          },
+        },
+        metadata: {
+          relations: {
+            viewer: { directly_related_user_types: [{ type: 'user' }] },
+          },
+        },
+      },
+      {
+        type: 'user',
+      },
+    ],
+  }}
+/>
+
+Relationship tuple with user `user:anne` or `user:3f7768e0-4fa7-4e93-8417-4da68ce1846c` may be written for objects with type `document` and relation `viewer`. Tuples with user `workspace:auth0` or `folder:planning#editor` will be rejected. This will affect only relations that are [directly related](#what-are-direct-and-implied-relationships) and have [the direct relationship keyword ("this")](./configuration-language.mdx#the-direct-relationship-keyword) in their relation definition.
```

Comment:
```suggestion
Relationship tuple with user `user:anne` or `user:3f7768e0-4fa7-4e93-8417-4da68ce1846c` may be written for objects with type `document` and relation `viewer`, so writing `{"user": "user:anne","relation":"viewer","object":"document:roadmap"}` will succeed.
Relationship tuple writes with user type that is not allowed for the `viewer` relation on objects of type `document`, for example `workspace:auth0` or `folder:planning#editor` will be rejected, so writing `{"user": "folder:product","relation":"viewer","object":"document:roadmap"}` will fail.
This will affect only relations that are [directly related](#what-are-direct-and-implied-relationships) and have [the direct relationship keyword ("this")](./configuration-language.mdx#the-direct-relationship-keyword) in their relation definition.
```